src/uct/base/uct_iface.c

```diff
@@ -452,6 +458,7 @@ UCS_CLASS_INIT_FUNC(uct_base_iface_t, uct_iface_ops_t *ops, uct_md_h md,
                        params->err_handler_flags : 0);
 
     self->md                = md;
+    self->internal_ops      = *internal_ops;
```

Comment:
Pointing instead of copying - done.

Regarding the second part - currently `uct_ib_iface_ops_t` derive from `uct_iface_ops_t`.
Do you mean deriving from `uct_iface_internal_ops_t` **instead** ?

Reply:
> Do you mean deriving from `uct_iface_internal_ops_t` **instead** ?

Yes. Now that we have internal ops, in makes more sens the TL-specific internal operations will derive from it, instead.